src/app/oauth2/middleware.js

```diff
@@ -15,6 +15,12 @@ module.exports = {
     next();
   },
 
+  setIpAddress: (req, res, next) => {
+    const ipAddress = req.headers["forwarded"].split(";")[0].split("=")[1];
+    req.session.ipAddress = ipAddress ? ipAddress : "unknown";
```

Comment:
More idiomatic for js would be `ipAddress || "unknown"` 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR#short-circuit_evaluation

Reply:
Good spot, fixed.